### PR TITLE
feat: choose the first value for the breadcrumb when an entity contains a list of values

### PR DESCRIPTION
**Pull request by Exac**
## Description
When provided multiple values for the breadcrumb, show the first one (for now).

### Testing
Please describe the tests that you ran to verify your changes. Please summarize what did you test and what needs to be tested e.g. deployed and tested helm chart locally. 

### Checklist:
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules

### Documentation
Make sure that you have documented corresponding changes in this repository or [hypertrace docs repo](https://github.com/hypertrace/hypertrace-docs-website) if required.

### CODE REVIEW

File: projects/observability/src/pages/apis/api-detail/api-detail-breadcrumb.resolver.ts [hunk 1]
```diff
@@ -99,10 +99,14 @@ export class ApiDetailBreadcrumbResolver<T extends EntityBreadcrumb> extends Ent
...
     }
+    const parentName: string | string[] = entity[this.getParentNameAttribute(parentTypeMetadata)] as string;
 
     return {
       parentId: entity[this.getParentIdAttribute(parentTypeMetadata)] as string,
-      parentName: entity[this.getParentNameAttribute(parentTypeMetadata)] as string
+      // TODO: (ENG-30212) How do we know which domain name to display when
+      //  there is an array? There is no guarantee that the first domain chosen
+      //  will match the domainId for the domain.
+      parentName: Array.isArray(parentName) ? parentName[0] : parentName
```
Comment (jake-bassett) [inline projects/observability/src/pages/apis/api-detail/api-detail-breadcrumb.resolver.ts:109]: Is there always guaranteed to be at least one element if there is an array?

In other words, do we need any null checking here?

**Review by jake-bassett — APPROVED** [review]
Commented, but approved if we don't need any null checks.

File: projects/observability/src/pages/apis/api-detail/api-detail-breadcrumb.resolver.ts (hunk 1 above)
Comment (Exac) [reply projects/observability/src/pages/apis/api-detail/api-detail-breadcrumb.resolver.ts:109]: Hmmn, that is a good point. We should simply return `{}` (like on line 100) if that were the case.

Comment (Exac) [reply projects/observability/src/pages/apis/api-detail/api-detail-breadcrumb.resolver.ts:113]: Added a check in 81e0269fed865201347ffba4bc5f44f8252d2c1b.

**Comment by Exac** [comment]
Going to close this for now, it shouldn't be necessary.

**Pull request closed without merging**